provision/testdata/enable_opflex_agent_prometheus.inp.yaml

```diff
@@ -0,0 +1,74 @@
+operator_managed_config:
+  enable_updates: true
+
+aci_config:
+  system_id: kube
+  use_legacy_kube_naming_convention: True
+  cluster_tenant: demo
+  apic_hosts:
+    - 10.30.120.100
+  apic_login:
+    username: admin
+    password: noir0123
```

Comment:
lets change this

Reply:
@snaiksat thanks for the catch, will get the dummy in picture..